### PR TITLE
tickets: relogin if needed

### PR DESCRIPTION
**Pull request by kaplun**
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

Closes https://sentry.cern.ch/inspire-sentry/inspire-labs/group/824604/

### CODE REVIEW

File: inspirehep/utils/tickets.py [hunk 1]
```diff
@@ -24,23 +24,36 @@
...
-from rt import ALL_QUEUES, ConnectionError
+from rt import ALL_QUEUES, ConnectionError, AuthorizationError
 from .proxies import rt_instance
 
 
 class EditTicketException(Exception):
     pass
 
+def relogin_if_needed(f):
+    """Repeat RT call after explicit login, if needed."""
```
Comment (david-caro) [inline inspirehep/utils/tickets.py]: s/login/login error/

Comment (david-caro) [inline inspirehep/utils/tickets.py]: Or something... not sure what you wanted to say there... xd

Comment (kaplun) [inline inspirehep/utils/tickets.py]: Nope, if the call of `f` fails, it will (re)login (in order to refresh the session), then rexecute `f`.

Comment (kaplun) [inline inspirehep/utils/tickets.py]: There's no involved login error. It's just a generic call to RT that fails due to expiration of something.

**Comment by kaplun** [comment]
@david-caro I have improved the explanation.

**Pull request merged**